craft_providers/lxd/lxc.py

```diff
@@ -614,7 +614,9 @@ def launch(  # noqa: PLR0912
                 )
                 logger.debug(str(error))
                 # Ignore first 3 failed attempts
-                if retry_count >= 2:
+                if retry_count >= 2 or (
```

Comment:
Can you update the `# Ignore first 3 failed attempts` comment also explain why we can ignore "create" in the error message?